src/main/java/com/drinchev/projectlabel/ProjectLabelStartupActivity.java

```diff
@@ -2,20 +2,14 @@
 
 import com.drinchev.projectlabel.resources.ui.ProjectLabelBackgroundImage;
 import com.intellij.openapi.project.Project;
-import com.intellij.openapi.startup.ProjectPostStartupActivity;
-import kotlin.Unit;
-import kotlin.coroutines.Continuation;
+import com.intellij.openapi.startup.StartupActivity;
 import org.jetbrains.annotations.NotNull;
-import org.jetbrains.annotations.Nullable;
 
-public class ProjectLabelStartupActivity implements ProjectPostStartupActivity {
```

Comment:
This was "experimental" and I couldn't pass the plugin validity check.

I tested it and works with `StartupActivity`.